src/elements/paginator/common/paginator-common.ts

```diff
@@ -0,0 +1,183 @@
+import { html, type LitElement, type PropertyValues, type TemplateResult } from 'lit';
+import { property } from 'lit/decorators.js';
+
+import { SbbLanguageController } from '../../core/controllers.js';
+import { hostAttributes } from '../../core/decorators.js';
+import { EventEmitter } from '../../core/eventing.js';
+import { i18nNextPage, i18nPreviousPage } from '../../core/i18n.js';
+import type { SbbPaginatorPageEventDetails } from '../../core/interfaces.js';
+import { type AbstractConstructor, SbbNegativeMixin } from '../../core/mixins.js';
+
+import '../../button/mini-button.js';
+import '../../button/mini-button-group.js';
+import '../../divider.js';
+
+export declare abstract class SbbPaginatorCommonElementMixinType {
+  public negative: boolean;
+  public length: number;
+  public pageSize: number;
+  public pageIndex: number;
+  public pagerPosition: 'start' | 'end';
+  public size: 'm' | 's';
+  protected language: SbbLanguageController;
+  protected numberOfPages(): number;
+  protected pageIndexChanged(value: number): void;
+  protected emitPageEvent(previousPageIndex: number): void;
+  protected renderPrevNextButtons(): TemplateResult;
+  protected abstract currentPageLabel(): string;
```

Comment:
this doesn't seem to be used in common interface. Is it really worth to have it here? shouldn't it be an internal of the two derived classes?

Reply:
if you mean `currentPageLabel()`, it's used in the common mixin in the `updated` hook at line 105

Reply:
Removed after Mursel review - the method from the 'normal' paginator was fine for the compact one too.